browser_patches/webkit/BUILD_NUMBER

```diff
@@ -1,2 +1,4 @@
 1499
 Changed: yurys@chromium.org Thu, Jun 10, 2021 11:01:06 PM
+1500
```

Comment:
This entire file should have been overwritten and contain just one entry. Did you use `./browser_patches/export.sh webkit` to prepare the PR?

Reply:
Edited it in place.

Reply:
I updated the file manually, but I thought I had change the committer name and date too. Next time I will use `./browser_patches/export.sh webkit`.